translations/en-us.yaml

```
@@ -18,6 +18,7 @@ component:
     edit-translation-text: Edit Translation
   locale-menu:
     select-your-preferred-language: Select your preferred language
-    locale-en-US: English (US) 
+    locale-en-US: English (US)
     locale-fr-FR: Français
     locale-pt-BR: Português (do Brasil)
+    locale-es: Spanish
```

@esbanarango Much thanks for adding the translations! To confirm, the translations are Spain-Spanish, right?

If so, maybe:

```suggestion
    locale-es: Español (de España)
```

If you meant a different locale, we'd want to update the YAML file names.

---

@ijlee2 These translations aren't really tied to any country. I tried to make them as neutral as possible.

---

Ah, ok. Sounds good to me, thanks!

I can probably get to reviewing the rest of your PR tonight/tomorrow.

---

Thank you @ijlee2 🙏